app/assets/javascripts/factories/repository/repository_data.js

```diff
@@ -0,0 +1,18 @@
+ManageIQ.angularApplication.factory('repositoryDataFactory', ['$http', function($http) {
+    var factoryRepository = {
```

Comment:
Factory here on purpose? This could also be a service (just `this.getRepositoryData = ...` instead of creating and returning an object), but not sure if you're planning to need the factory flexibility later..?